pom.xml

```diff
@@ -159,6 +160,7 @@
                     <archive>
                         <manifest>
                             <mainClass>com.exasol.bucketfs.client.BucketFsClient</mainClass>
+                            <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
```

Comment:
👍 
Would it make sense to move this to PK?

Reply:
See https://github.com/exasol/project-keeper/issues/427